src/axom/slic/interface/slic_macros.hpp

```diff
@@ -111,22 +127,29 @@
     }                                                                 \
   } while(axom::slic::detail::false_value)
 
-/// @}
+///@}
 
 // Use complete debug macros when not on device
 #if defined(AXOM_DEBUG) && !defined(AXOM_DEVICE_CODE)
 
   //-----------------------------------------------------------------------------
-  /// \name ASSERT MACROS
   /// @{
+  //! \name ASSERT MACROS
+  //!
+  //! \attention These assert macros are collective operations.
+  //! All ranks in the user-supplied communicator must call the macro
+  //! when used within an MPI distributed environment, and abort on error
+  //! messages is enabled for the current active logger (default is enabled
+  //! for loggers)
+  //!
 
   /*!
  * \def SLIC_ASSERT( EXP )
```

Comment:
This macro is not showing up properly in the doxygen:

https://axom.readthedocs.io/en/docs-han12-slicmpi/doxygen/html/slic__macros_8hpp.html#ad99a3eb01f4e3abe97a4ce625095f64e

Along with others.  It looks like the doxygen build needs `AXOM_DEBUG` enabled.